examples/pii_masking_example/llamaindex_agentic_fastapi/app_presidio.py

```
@@ -0,0 +1,408 @@
+subscription_key = os.environ["AZURE_SUBSCRIPTION_KEY"]
+model = "gpt-4o-mini"
+
+FI_LLM = AzureOpenAI(
+    azure_endpoint=endpoint,
+    model = model,
+    api_key=subscription_key,
+    api_version="2024-05-01-preview",
+    engine=deployment
+)
+import random
+from fastapi import FastAPI, BackgroundTasks
```

Ruff (0.8.2)

Redefinition of unused `FastAPI` from line 8

Remove definition: `FastAPI`

(F811)